package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
A matter of taste, but I think we can avoid this `package.json` file, and stick to calling this manually, or keeping `execute-tests`.

In fact, Travis now supports Elm - https://docs.travis-ci.com/user/languages/elm

Reply:
This is where the TB should guide us. That is, if time will permit, we can use. However, I'd say - to be on the safe side - lets revert to the existing Travis, and open a follow up issue to update Travis.